src/platform-implementation-js/dom-driver/gmail/views/gmail-route-view/gmail-route-view.ts

```diff
@@ -460,7 +460,7 @@ class GmailRouteView {
     let threadContainerElement: HTMLElement | 'destroyed';
 
     const selector = 'table.Bs > tr';
-    const selector_2023_11_16 = '.ao8:has(.a98.iY), .ao9:has(.apa)';
+    const selector_2023_11_16 = '.ao9:has(.a98.iY), .ao9:has(.apa)';
```

Comment:
```suggestion
    const selector_2023_11_16 = '.ao9:has(.a98.iY, .apa)';
```

We may want to update the date in the variable name.

Reply:
it is same date, choosing .ao8 was a mistake (altho we couldn't repro ourselves so we didn't know about mutation observer issue)

Reply:
the suggested change is not much different from existing code, will keep as is